taca_ngi_pipeline/cli.py

```diff
@@ -142,3 +152,26 @@ def _exec_fn(obj, fn):
         else:
             logger.error("processing {} failed - reason: {}, operator {} has been notified".format(
                 str(obj), str(e), obj.config.get('operator')))
+
+
+@deliver.command()
+@click.pass_context
+@click.argument('projectid', required=False, default=None)
+def check_status(context, projectid=None):
+    import pdb; pdb.set_trace()
+    # how do we access config file??
+    stagingpathhard = CONFIG.get('deliver', {}).get('stagingpathhard')
+
+    if stagingpathhard is None:
+        logger.error('Config file requires "stagingpathhard"!!')
+        exit(1)
+
+    # doing stupid stuff - creating whatever random project, just to get the path to DELIVERY_HARD
+    stagingpathhard = _deliver.ProjectDeliverer('P4601').expand_path(stagingpathhard)
+    stagingpathhard = os.path.abspath(os.path.join(stagingpathhard, '..'))
+
```

Comment:
I haven't found a proper way to get `stagingpathhard`, so this needs to be re-implemented in a better way